src/main/java/com/faforever/client/main/MainController.java

```diff
@@ -101,10 +103,13 @@
 @Slf4j
 // TODO divide and conquer
 public class MainController implements Controller<Node>, InitializingBean {
+
   private static final PseudoClass NOTIFICATION_INFO_PSEUDO_CLASS = PseudoClass.getPseudoClass("info");
   private static final PseudoClass NOTIFICATION_WARN_PSEUDO_CLASS = PseudoClass.getPseudoClass("warn");
   private static final PseudoClass NOTIFICATION_ERROR_PSEUDO_CLASS = PseudoClass.getPseudoClass("error");
   private static final PseudoClass HIGHLIGHTED = PseudoClass.getPseudoClass("highlighted");
+  private final static ReadOnlyObjectWrapper<NavigationItem> CURRENT_TAB = new ReadOnlyObjectWrapper<>();
+
```

Comment:
Why is this static it doesn't really make any sense to be static since it is a property of a specific instance not shared across instances

Reply:
I meant you could make it a property of the MainController rather than static

Reply:
It won't help, it won't be possible to take him from another class. I have already found another solution. https://github.com/FAForever/downlords-faf-client/pull/2609/commits/c29b16ea198628d05933fa4927183811bc620749